src/postgres/commands/createPostgresServer/steps/PostgresServerCreateStep.ts

```diff
@@ -16,7 +16,8 @@ export class PostgresServerCreateStep extends AzureWizardExecuteStep<IPostgresSe
 
     public async execute(wizardContext: IPostgresServerWizardContext, progress: Progress<{ message?: string; increment?: number }>): Promise<void> {
 
-        const locationName = nonNullProp(nonNullProp(wizardContext, 'location'), 'name');
+        /* eslint-disable @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-assignment */
```

Comment:
You shouldn't have to disable all this eslint stuff. Maybe VS Code just hadn't realized you updated the 'vscode-azureextensionui' package yet. Sometimes you have to wait a bit or reload the window after updating dependencies for warnings to go away

Reply:
Ok yea, I think I will try that. I was actually surprised why I was getting so many lint errors.